in_toto/models/layout.py

```diff
@@ -148,58 +147,67 @@ def import_step_metadata_from_files_as_dict(self):
   def _validate_type(self):
     """Private method to check that the type string is set to layout."""
     if self._type != "layout":
-      raise FormatError("Invalid _type value for layout (Should be 'layout')")
+      raise securesystemslib.exceptions.FormatError(
+          "Invalid _type value for layout (Should be 'layout')")
 
   def _validate_expires(self):
     """Priavte method to verify the expiration field."""
     try:
       date = parse(self.expires)
-      in_toto.ssl_crypto.formats.ISO8601_DATETIME_SCHEMA.check_match(self.expires)
-    except:
-      raise FormatError("Malformed date string in layout!")
+      securesystemslib.formats.ISO8601_DATETIME_SCHEMA.check_match(
+          self.expires)
+    except Exception:
+      raise securesystemslib.exceptions.FormatError(
+          "Malformed date string in layout!")
 
   def _validate_keys(self):
     """Private method to ensure that the keys contained are right."""
     if type(self.keys) != dict:
-      raise FormatError("keys dictionary is malformed!")
+      raise securesystemslib.exceptions.FormatError(
+          "keys dictionary is malformed!")
 
-    in_toto.ssl_crypto.formats.KEYDICT_SCHEMA.check_match(self.keys)
+    securesystemslib.formats.KEYDICT_SCHEMA.check_match(self.keys)
 
     for keyid, key in six.iteritems(self.keys):
       if 'private' in key and key['private'] != '':
-        raise FormatError("key: {} contains a private key part!".format(keyid))
+        raise securesystemslib.exceptions.FormatError(
+            "key: {} contains a private key part!".format(keyid))
```

Comment:
I updated to 0.10.4 in https://github.com/in-toto/in-toto/pull/65/commits/df25016ff32603935788b5a662d648a94256609e.
Unfortunately I can't rename the branch without loosing this GitHub discussion but I updated the PR title.